### ISSUE TITLE
Language button layout is not ok on Safari browser

### ISSUE THREAD

**Issue by timothyTheCoder** [XRPLF/xrpl-dev-portal#995]
The layout of the language button on the top right corner is all messed up on Safari on a Mac. It is also not ok on iPad.

**Comment by mDuo13** [comment]
Verified that this is the case on my roommate's Macbook, but haven't yet figured out the cause. Some kind of CSS incompatibility, presumably.

**Comment by timothyTheCoder** [comment]
Created pull request https://github.com/ripple/xrpl-dev-portal/pull/1001

**Comment by timothyBrake** [comment]
I have made the changes and also another issue related to outline of the menu items on Safari:
- When clicking on a menu item, the default outline border is visible only on Safari. 
- The jump-to-top has a yellow background and white letters, which makes reading it almost impossible. Changed it to transparent.

I am unable to push to a remote branch though. Any idea where to write access to new branches?
